vng_api_common/filters.py

```diff
@@ -86,14 +91,17 @@ def url_to_pk(self, url: str):
         return instance.pk
 
     def to_python(self, value: str):
-        # TODO: validate that it's proper URL input
+        if value is not None:
+            try:
+                URLValidator()(value)
+            except ValidationError as exc:
+                raise exc
+
         if value:
             try:
                 value = self.url_to_pk(value)
             except models.ObjectDoesNotExist:
-                raise ValidationError(
-                    _("Invalid resource URL supplied"), code="invalid"
-                )
+                return []
```

Comment:
Can you please add a `logger.info(...)` call? We will definitely get support questions about this.